webui/react/src/components/kit/Avatar.tsx

```diff
@@ -10,19 +10,24 @@ import css from './Avatar.module.scss';
 
 export const Size = {
   ExtraLarge: 'extra-large',
+  ExtraSmall: 'extra-small',
   Large: 'large',
   Medium: 'medium',
+  Small: 'small',
 } as const;
 
 export type Size = ValueOf<typeof Size>;
 
 export interface Props {
-  displayName: string;
+  text: string;
   hideTooltip?: boolean;
-  /** do not color the bg based on displayName */
+  /** do not color the bg based on text */
   noColor?: boolean;
```

Comment:
I don't think we use this props, right?

Reply:
this actually is from the version of `Avatar` that was shared with SaaS, which does use it.